.changeset/clever-keys-hammer.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/mdx': patch
```

Comment:
Should this be a minor?
```suggestion
'@astrojs/mdx': minor
```

Reply:
Since this integration is still pre-1.0, we're treating patches as "minors" and minors as "majors." Good comment though 👍

Reply:
Ah right didn't notice it was pre 1.0 👍